pkg/models/task.go

```diff
@@ -62,6 +64,7 @@ func NewTaskId() string {
 }
 
 type TaskDirective struct {
+	NotificationId     string
```

Comment:
There is notificationId in Task, is it needed here?

Reply:
yes,will use to write socket message.